repository/src/main/java/ru/karalionak/rest/dao/impl/CertificateDaoImpl.java

```diff
@@ -70,37 +70,27 @@ public int getBatchSize() {
     }
```

Comment:
Try if this works:
`    @Override
    public void addTags(long certificateId, List<Long> tagIds) {
        final BeanPropertySqlParameterSource[] parameters = tagIds.stream()
                .map(BeanPropertySqlParameterSource::new)
                .toArray(BeanPropertySqlParameterSource[]::new);
        namedJdbcTemplate.batchUpdate(CREATE_REFERENCE_BETWEEN_CERTIFICATE_TAG, parameters);
    }`